procedures/lvv-t1992.ipynb

```diff
@@ -20,239 +20,277 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
     "from lsst.ts import salobj\n",
-    "from lsst.ts.idl.enums import Hexapod\n",
+    "from lsst.ts.idl.enums import MTHexapod\n",
     "import logging\n",
     "import asyncio\n",
-    "import os"
+    "import os\n",
+    "import yaml\n",
+    "\n",
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt\n",
+    "from astropy.time import Time\n",
+    "from datetime import datetime, timedelta\n",
+    "import pandas as pd\n",
+    "import astropy.units as u"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following block sets the necessary environment variables for setting up the DDS/SAL communication"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "os.environ[\"LSST_DDS_DOMAIN\"] = 'citest'\n",
-    "os.environ[\"OSPL_URI\"] = \"file:///scratch/tribeiro/ospl.xml\"\n",
+    "#os.environ[\"LSST_DDS_DOMAIN\"] = 'citest'\n",
```

Comment:
When would you use this? Perhaps delete it?

A sync time of 10 seconds is too short. If historical data arrives after that it will be ignored during SalInfo startup and will show up as new data -- and worse, it'll be intermixed with new data in a way that is impossible to tell which is which without looking at the private_sndStamp field. Short timeouts are dangerous!

Reply:
Updated to  30 sec. Enough?